src/drtsans/tof/eqsans/elastic_reference_normalization.py

```diff
@@ -159,7 +159,7 @@ def calculate_elastic_reference_normalization(wl_vec, q_vec, ref_i_of_q):
 
     """
     # Reshape Q, wavelength, intensities and errors to unique 1D array or 2D array
-    _, _, ref_i_array, ref_error_array, _ = reshape_q_wavelength_matrix(ref_i_of_q)
+    _, _, ref_i_array, ref_error_array, _ = reshape_q_wavelength_matrix(ref_i_1d)
```

Comment:
is it always `q_wavelength` or can it also be `phi_wavelength`?

Reply:
It can also be `phi_wavelength`. Any suggestion for what it should be named?
These are my candidates, but I am not a huge fan of any of them :slightly_smiling_face: 
- `reshape_q_wavelength_matrix` (keep existing name)
- `reshape_q_or_phi_wavelength_matrix`
- `reshape_x_wavelength_matrix`

Reply:
`(Q, w)` or `(phi, w)` are both the "domain" of the intensity values. We could use this word, for instance `reshape_intensity_domain()`